Ivan.html

```diff
@@ -1 +1,26 @@
 
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>page2</title>
+</head>
+<body>
+    <h1>Druga strana: idemoooooo!!!</h1>
+    <p>
```

Comment:
NE MOZE TO TAKO!!!!

Reply:
Porke No?!

Reply:
his branch has conflicts that must be resolved

Reply:
AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA!!!!!

Reply:
💃